src/styles/utils/variables.scss

```diff
@@ -0,0 +1,2 @@
+@import './cycles';
+@import '../index';
```

Comment:
It's not a good practice to import other SCSS files into your variables file. The variables file should only contain variables, mixins, or placeholders that can be imported into other files without causing side effects like duplicating styles or outputting CSS. You should remove the import of '../index' from here.